css/templates/land-see/page.css

```diff
@@ -279,3 +279,89 @@ h4,
     justify-content: flex-end;
   }
 }
+
+/* Subscribe */
+
+.land-see-subscribe {
+  position: relative;
+  height: 310px;
+  overflow: hidden;
+  color: var(--color-font-footer);
+}
+
+.land-see-subscribe amp-img {
+  position: absolute;
+  width: 100%;
+  object-position: top center;
+}
+
+amp-img.cover img {
+  object-fit: cover;
+}
+
+.land-see-subscribe-bg {
+  z-index: 1;
+}
+
+.land-see-subscribe-form {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 2;
+}
+
+.land-see-subscribe-form form {
+  padding: 0;
+}
+
+.land-see-subscribe-form h2 {
+  margin-bottom: var(--h3);
+  padding-top: var(--h1);
+}
+
+.land-see-subs-heading-text {
+  font-size: var(--h2);
+  position: relative;
+}
+
+.land-see-subs-heading-text:after {
+  content: 'Land+See';
+  font-weight: bold;
+}
+
+.ampstart-input {
+  margin-bottom: 0;
+}
+
+.ampstart-input > label {
+  display: none;
+}
+
+.ampstart-input input:not([type=checkbox]):not([type=radio]) {
+  -webkit-appearance: none;
+}
+
+.ampstart-input input[type="email"] {
+  border: 2px solid var(--color-font-secondary);
+  padding: 2px 0 0 var(--h9);
+  font-size: var(--h8);
+  color: var(--color-font-footer);
+}
+
+.ampstart-input input[type="submit"] {
```

Comment:
are we modifying all the submit buttons on the template? That does not sound right

Reply:
Good call. We only have one submit input, for now, but I'll scope styles to the subscription form.